script/build

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
That seems pretty reasonable, I'm also not super sure what the ideal process is here. Perhaps at some point we make a `script/release` in all the agents that does the necessary stuff. I think for now we should include a Release section in the README like we do on the DR agents. Sounds reasonable to you?